Tutorial.md

```diff
@@ -142,7 +133,7 @@ svtools copynumber \
 ```
 **Note:** The argument to the `--cnvnator` option of `svtools copynumber` may need to be the full path to the cnvnator-multi executable included as part of SpeedSeq. This example assumes cnvnator-multi is installed system-wide. 
 
-### Use `svtools vcfpaste` to construct a VCF that pastes in genotype and copynumber information
+### Use `svtools vcfpaste` to construct a VCF that pastes together the individual genotyped and copy-number annotated vcfs
```

Comment:
I was preferring copynumber to Copy Number or copy-number. We should be consistent.